scripts/sct_deepseg.py

```diff
@@ -143,26 +173,46 @@ def main(argv):
             if not deepseg.models.is_valid(path_model):
                 parser.error("The input model is invalid: {}".format(path_model))
 
+        # Order input images
+        if args.c is not None:
+            input_filenames = []
+            for required_contrasts in deepseg.models.MODELS[name_model]['contrasts']:
+                for provided_contrasts, input_filename in zip(args.c, args.i):
+                    if required_contrasts == provided_contrasts:
```

Comment:
```suggestion
            for required_contrast in deepseg.models.MODELS[name_model]['contrasts']:
                for provided_contrast, input_filename in zip(args.c, args.i):
                    if required_contrast == provided_contrast:
```
Am I being picky? simply because `required_contrasts` variable already exists.. :3